setup/configuration.py

```diff
@@ -0,0 +1,133 @@
+import os
+import subprocess
+import platform
+
+from sre_constants import SRE_FLAG_DEBUG
+from django.contrib.auth.models import User
+from player.models import Player
+from player.models import PlayerData
+from faction.models import Faction
+from setup.models import APIKey
+from awards.models import AwardsData
+from faction.models import FactionData
+from bazaar.models import BazaarData
+from company.models import CompanyDescription
+from loot.models import NPC
+from bot.models import Bot
+from decouple import config
+
+class Configuration:
+    reset_db_question = "Do you want to reset the database?"
+    fill_db_question = "Do you want to fill the database?"
+    key = config('SECRET_KEY')
+
+    def question(self, question):
+        reply = str(input(question + ' (y/n): ')).lower().strip() 
+
+        confirmation = [ "yes", "ye", "y" ]
+
+        if (reply in confirmation or reply in [ 'no', 'n']):
+            return reply in confirmation
+        else:
+            return self.question("Invalid input. Please enter")
+
+    @staticmethod
+    def executeCommand(command):
+        try:
+            if command[:2] == "rm" and platform.system() == "Windows":
+                subprocess.run(["C:\Program Files\Git\git-bash.exe", "c", command], check = True, shell = True)
```

Comment:
Windows doesn't support rm for deleting things, git bash for windows does however